src/react/components/pages/homepage/homePage.tsx

```diff
@@ -44,10 +56,29 @@ export default class HomePage extends React.Component<IHomepageProps> {
     public state: IHomepageState = {
         cloudPickerOpen: false,
     };
-
     private filePicker: React.RefObject<FilePicker> = React.createRef();
     private deleteConfirm: React.RefObject<Confirm> = React.createRef();
     private cloudFilePicker: React.RefObject<CloudFilePicker> = React.createRef();
+    private importConfirm: React.RefObject<Confirm> = React.createRef();
+
+    constructor(props: IHomepageProps, context) {
```

Comment:
You should be able to do this without the constructor. You can define state inline like:
```
export default class HomePage extends React.Component<IHomepageProps> {

    state = {
        cloudPickerOpen: false
    }
```
Also, on lines 59-62, you're already creating the Ref instance, so you don't need the creation inside the constructor. Nit: you can also just add the type in that first instantiation

And for the binding of functions, since you're using `function x = () => {` syntax, it automatically binds for you :)

Reply:
Got it! I had added the constructor to circumvent an issue that was solved by another change so I switched it back to how it was before.